force-app/main/fslStatusTransition/classes/HOT_WorkOrderLineItemStatusHandler.cls

```diff
@@ -137,8 +137,9 @@ public without sharing class HOT_WorkOrderLineItemStatusHandler extends MyTrigge
                 if (numberOfScheduled > 0) {
                     workOrderLineItem.Status = 'Scheduled';
                 }
-                if (numberOfDispatched > 0) {
+                if (numberOfDispatched == numberOfServiceAppointments) {
                     workOrderLineItem.Status = 'Dispatched';
+                    continue;
```

Comment:
Du kan slette alt som har med numberOfPartiallyComplete i denne klassen da den statusen ikke finnes på SA, så blir det litt mer oversiktlig.

Reply:
```
 if (numberOfCannotComplete > 0) {
      workOrder.Status = 'Partially Complete';
}
```

Her må du kanskje endre til `numberOfCannotComplete > 0 && (numberOfCannotComplete + numberOfCompleted) =  numberOfWorkOrderLineItem` eller noe så den kun blir satt til Partially Complete om alle er i en sluttstatus.

Blir vel samme for den under med `numberOfCompleted > 0`.